.jenkins.yml

```diff
@@ -611,10 +615,6 @@ exclude:
     target: swiftpm
     configuration: Release
 
-  - xcode_version: 12.0
```

Comment:
Could you explain why this is being removed but 11.3 added?

Reply:
This is the excludes section, so it's disabling building on 11 and enabling building on 12.